PG.StarWarsGame.Files.MEG/PG.StarWarsGame.Files.MEG/Binary/File/Builder/V1/MegFileBuilder.cs

```diff
@@ -42,6 +46,61 @@ public MegFile FromBytes(byte[] byteStream)
             return new MegFile(header, megFileNameTable, megFileContentTable);
         }
 
+        public MegFile FromHolder(MegFileHolder holder, out List<string> filesToStream)
+        {
+            List<string> files = holder.Content.Select(megFileDataEntry => megFileDataEntry.RelativeFilePath).ToList();
+            List<MegFileNameTableRecord> megFileNameTableRecords =
+                files.Select(file => new MegFileNameTableRecord(file)).ToList();
+            megFileNameTableRecords.Sort();
+            // Workaround for Unix compatibility.
+            // File names are always stored as uppercase and without delimiter (\0), but Unix's file system is case sensitive,
+            // so we cache the proper file paths sorted by the "cleaned" version's CRC for quick access later.  
+            filesToStream = new List<string>();
```

Comment:
Would we break stuff if we enable Unix workarounds for a Unix environment only, using `RuntimeInformation.IsUnix()`?

Reply:
Seen as the IDictionary input allows arbitrary file names which dictate the order it's no longer a Unix-only thing, but necessary as a whole, otherwise we can't determine the order o the files to stream into the archive. By now it's more a cache of files to pack in a given order than just a Unix workaround.
I'll remove the comment.